Diagnostic/lad_config_all.py

```diff
@@ -432,7 +432,7 @@ def generate_all_configs(self):
             fileLogs_setting = self._ext_settings.get_fileLogs_setting()
             lad_logging_config_helper = LadLoggingConfig(syslogEvents_setting, fileLogs_setting, self._sink_configs,
                                                          self._pkey_path, self._cert_path, self._encrypt_secret)
-            mdsd_syslog_config = lad_logging_config_helper.get_mdsd_syslog_config()
+            mdsd_syslog_config = lad_logging_config_helper.get_mdsd_syslog_config(self._ext_settings.read_protected_config('disableStorageAccount') == True)
```

Comment:
where is disableStorageAccount to be defined? In public json?

Reply:
under protected settings. like so - 

{
	"protectedSettings": {
		"storageAccountName": "__PlaceholderStorageAccountName__",
		"storageAccountSasToken": "__PlaceholderSASToken__",
		**"disableStorageAccount": true,**
		"sinksConfig": {
			"sink": [
				{
					"name": "IdleEventHubSink",
					"type": "EventHub",
					"sasURL": "<...>"
				}
			]
		}
	}
}